src/App/Fossa/Main.hs

```diff
@@ -84,14 +85,27 @@ appMain = do
           }
 
   case optCommand of
+    AnalyzeCommand AnalyzeOptions{analyzeOutput, analyzeBranch, analyzeMetadata, monorepoAnalysisOpts = (MonorepoAnalysisOpts (Just monorepoAnalysisType)), analyzeBaseDir} -> do
+      dieOnWindows "Monorepo analysis is not supported on Windows"
+      if analyzeOutput
+        then die "Monorepo analysis does not support stdout scan destination"
+        else do
+          key <- requireKey maybeApiKey
+          let apiOpts = ApiOpts optBaseUrl key
+          let metadata = maybe analyzeMetadata (mergeFileCmdMetadata analyzeMetadata) fileConfig
+          let monorepoAnalysisOpts = MonorepoAnalysisOpts (Just monorepoAnalysisType)
```

Comment:
[nit] the pattern match above should already bind `monorepoAnalysisOpts` to this value, I think(?)

Reply:
It doesn't seem to do so:
```
Couldn't match expected type ‘MonorepoAnalysisOpts’
              with actual type ‘AnalyzeOptions -> MonorepoAnalysisOpts’
```

And I can't access `AnalyzeOptions` because we're deconstructing it in the pattern match statement.

Reply:
weird. there's syntax we can use to bind the `AnalyzeOptions` as part of the pattern match:

```hs
opts@AnalyzeOptions{ etc }
```

though I'm not sure that would make this clearer. oh well